### DISCUSSION TITLE
Item alternative name needed

### DISCUSSION THREAD

**Discussion by alsaihati** [Blair2004/NexoPOS#305] [Ideas]
Hi

Please 
1.add alternative item name I nexo 4
2. Arabia tax 
3. Detail emailable

**Comment by Blair2004** [comment]
Not sure to understand all the points.

**Comment by alsaihati** [comment]
1. Alternative item name:
In add product page need new field with Alternative name for different language name.
2. Arabic tax: the tax calculation that already gave you before.
3. Detail report need to add send email feature 
Thank you
